### PR TITLE
tp-qemu: mac_change: disable SLES.11.4 guest for up_change.

### PR DESCRIPTION
**Pull request by CongSmile**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1263150

### CODE REVIEW

File: generic/tests/cfg/mac_change.cfg [hunk 1]
```diff
@@ -13,7 +13,7 @@
         - @up_change:
             virtio_net:
                 no RHEL.5, RHEL.6.0, RHEL.6.1, RHEL.6.2, RHEL.6.3
-                no SLES.10.4 SLES.11.0 SLES.11.1 SLES.11.2 SLES.11.3
+                no SLES.10.4 SLES.11.0 SLES.11.1 SLES.11.2 SLES.11.3 SLES.11.4
```
Comment (suqinhuang) [inline generic/tests/cfg/mac_change.cfg]: Hi coli,
how about disable SLES directly

Comment (CongSmile) [inline generic/tests/cfg/mac_change.cfg]: We can't do that.

As SLES.12 support it and maybe other enterprises support it too.

Thanks.

Comment (suqinhuang) [inline generic/tests/cfg/mac_change.cfg]: ok, forgot there is sles.12

Comment (suqinhuang) [inline generic/tests/cfg/mac_change.cfg]: Acked-by: Suqin Huang

**Comment by ypu** [comment]
Hi @CongSmile 
Agree with @suqinhuang we may should make it short than current line. Maybe SLES.10 and SLES.11 can cover the request.

**Comment by CongSmile** [comment]
@ypu 

Yes, I can get your meaning.

We can disable all SLES.10 for the case, as SLES.10.4 is the last one.
My concern is that I'm not sure whether the other new versions support it any more in the future.
e.g. SLES.11.5 (not sure SLES.11.4 is the last one).

Thanks.

**Comment by ypu** [comment]
@CongSmile OK That make sense. So
Acked-by: Yiqiao Pu <ypu@redhat.com>

**Comment by ypu** [comment]
Thanks @CongSmile and @suqinhuang will merge it.

**Pull request merged**